libwallet/assets/dcr/dex_wallet.go

```diff
@@ -18,9 +18,15 @@ import (
 	dexasset "decred.org/dcrdex/client/asset"
 	dexdcr "decred.org/dcrdex/client/asset/dcr"
 	"decred.org/dcrdex/dex"
-	walleterrors "decred.org/dcrwallet/v3/errors"
+
+	// Version v3 of dcrwallet library is still in use because dcrdex is yet to upgrade
+	// to dcrd v2.
```

Comment:
you mean v4?

Reply:
v4 is the tag but v2.0.2 is the release version. I will use v4 instead.